web/src/common/theme.js

```diff
@@ -18,10 +18,11 @@ const theme = createTheme({
       darkPurple: "#260B34",
     },
     text: {
+      title: "#955ab5",
       subtitle: "#808080",
       // For less essential body text that we don't want to stand out as much.
       secondaryBody: "#666666",
-      textButton: "#955ab5",
+      textButton: "#260b34",
```

Comment:
Should we revert this? This value is used for the highlighting of the nav bar and for the "View on Facebook" button in the event card. There's not enough contrast between the dark purple and the black text